rocketmq-flink/README.md

```diff
@@ -0,0 +1,147 @@
+        consumerProps.setProperty(RocketMqConfig.CONSUMER_GROUP, "c002");
+        consumerProps.setProperty(RocketMqConfig.CONSUMER_TOPIC, "flink-source2");
+
+        Properties producerProps = new Properties();
+        producerProps.setProperty(RocketMqConfig.NAME_SERVER_ADDR, "localhost:9876");
+
+        env.addSource(new RocketMQSource(new SimpleKeyValueDeserializationSchema("id", "address"), consumerProps))
+            .name("rocketmq-source")
+            .setParallelism(2)
+            .process(new ProcessFunction<Map, Map>() {
+                @Override
+                public void processElement(Map in, Context ctx, Collector<Map> out) throws Exception {
+                    HashMap result = new HashMap();
+                    result.put("id", in.get("id"));
+                    String[] arr = in.get("address").toString().split("\\s+");
+                    result.put("province", arr[arr.length-1]);
+                    out.collect(result);
+                }
+            })
+            .name("upper-processor")
+            .setParallelism(2)
+            .addSink(new RocketMQSink(new SimpleKeyValueSerializationSchema("id", "province"),
+                new DefaultTopicSelector("flink-sink2"), producerProps).withBatchFlushOnCheckpoint(true))
+            .name("rocketmq-sink")
+            .setParallelism(2);
+
+        try {
+            env.execute("rocketmq-flink-example");
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+ ```
+
+## Configurations
+The following configurations are all from the class `org.apache.rocketmq.flink.RocketMqConfig`.
+
+### Producer Configurations
+| NAME        | DESCRIPTION           | DEFAULT  |
+| ------------- |:-------------:|:------:|
+| nameserver.address      | name server address *Required* | null |
+| nameserver.poll.interval      | name server poll topic info interval     |   30000 |
+| brokerserver.heartbeat.interval | broker server heartbeat interval      |    30000 |
+| producer.group | producer group      |    $UUID |
+| producer.retry.times | producer send messages retry times      |    3 |
+| producer.timeout | producer send messages timeout      |    3000 |
+
+
+### Consumer Configurations
+| NAME        | DESCRIPTION           | DEFAULT  |
+| ------------- |:-------------:|:------:|
+| nameserver.address      | name server address *Required* | null |
+| nameserver.poll.interval      | name server poll topic info interval     |   30000 |
+| brokerserver.heartbeat.interval | broker server heartbeat interval      |    30000 |
+| consumer.group | consumer group *Required*     |    null |
+| consumer.topic | consumer topic *Required*       |    null |
+| consumer.tag | consumer topic tag      |    * |
+| consumer.offset.reset.to | what to do when there is no initial offset on the server      |   latest/earliest/timestamp |
+| consumer.offset.from.timestamp | the timestamp when `consumer.offset.reset.to=timestamp` was set   |   $TIMESTAMP |
```

Comment:
The default value $TIMESTAMP and $UUID are confusing, especially the $ symbol, a short description may be better.

Reply:
Good catch. Will fix.